web/modules/custom/dpl_fees/dpl_fees.module

```diff
@@ -19,3 +21,57 @@ function dpl_fees_dpl_react_apps_data(array &$data): void {
     ),
   ];
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * @param array<mixed> $form
+ *   An associative array containing the structure of the form.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The current state of the form.
+ */
+function dpl_fees_form_dpl_library_agency_list_size_settings_form_alter(array &$form, FormStateInterface $form_state): void {
+  $config = \Drupal::service('config.factory')
+    ->getEditable('dpl_library_agency.list_size_settings');
```

Comment:
You should use config service here instead.

Reply:
Are now using config service instead of config factory.